LICENCE

```diff
@@ -0,0 +1,642 @@
+This program and the accompanying materials are made available under the
```

Comment:
I was wondering: the content of LICENSE looks like markdown, so should we rename the file to reflect that? Here since we introduce it, it's a good opportunity, if we want to do so. The Eclipse Foundation explicitly say we are allowed to do exactly this.